documentation/2.0/content/userguide/tools/deploy.md

```diff
@@ -41,3 +41,24 @@ If the model or variables file contains passwords encrypted with the WDT Encrypt
 ### Using multiple models
 
 The Deploy Applications Tool supports the use of multiple models, as described in [Using multiple models]({{< relref "/concepts/model#using-multiple-models" >}}).
+
+## Parameter table for deployApps
+
+| Parameter | Definition | Default |
+| ---- | ---- | ---- |
+| -admin_pass_env | An alternative to entering the admin password at a prompt. The value is an ENVIRONMENT VARIABLE name that WDT will use to retrieve the password. |    |
+| -admin_pass_file |  An alternative to entering the admin password at a prompt. The value is a the name of a file that contains a password string that the tool will read to retrieve the password. |    |
+| -admin_url | The admin server URL used for online deploy. |    |
+| -admin_user | The admin username used for online deploy. |    |
+| -archive_file | The path to the archive file. If the model_file argument is not used, the model file in this file will be used. This can also be specified as a comma-separated list of archive files.  The overlapping contents in each archive take precedence over previous archives in the list. |    |
+| -cancel_changes_if_restart_required | Cancel the changes if the update requires domain restart. |    |
+| -discard_current_edit | Discard all current edits before starting update. |    |
+| -domain_home | (Required). The domain home directory. |    |
+| -domain_type | The type of domain. | WLS |
+| -model_file | The location of the model file. This can also be specified as a comma-separated list of model locations, where each successive model layers on top of the previous ones. |    |
+| -oracle_home | The location of the existing oracle home directory. Required if ORACLE_HOME environment variable is not set. |    |
```

Comment:
oracle home -> Oracle home (I see both Oracle home and Oracle Home in the docs. We should select one choice and use it consistently.)